src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventCounter.cs

```diff
@@ -83,14 +88,13 @@ internal override void WritePayload(float intervalSec)
             lock (MyLock)
             {
                 Flush();
-                EventCounterPayload payload = new EventCounterPayload();
-                payload.Name = _name;
+                CounterPayload payload = new CounterPayload();
                 payload.Count = _count;
                 payload.IntervalSec = intervalSec;
                 if (0 < _count)
```

Comment:
I think this condition is correct for computing the `Mean`, but it should be `_count > 1` for standard deviation.